### DISCUSSION TITLE
Reading CSVF UVWFILE file

### DISCUSSION THREAD

**Discussion by Amilwickz** [firemodels/fds#12999] [Q&A] [answered]
Hello,

I am using FDS-6.8.0-0-g886e009-release. In my simulation I have 4 meshes.

I have included the following line to save the velocity profile time to time and I can see those files are created with no issue.
&DUMP DT_UVW=30.0,DT_UVW=50.0,DT_UVW=70.0, /

When I tried to import these created velocity profile (only the DT_UVW=70.0) back in a new simulation using,
&CSVF UVWFILE='CSV_test_uvw_t2_m1.csv'/
I get the following error:
**ERROR: Problem with MESH 1; UVW_FILE CSV_test_uvw_t2_m1.csv1.csv does not exist. (MPI Process: 0, CHID: CSV_test)**

Note 1: I have put all the created CSV files in the working folder of the new simulation.

Note 2: My first mesh is different from the latter three meshes in terms of 'X' distance. See below:
&MESH IJK=5,10,10       XB=10.0,20.0,              0.0,20.0,0.0,20.0/ 	 2 m MESH 1
&MESH IJK=10,10,10      XB=20.0,40.0,             0.0,20.0,0.0,20.0/ 	 2 m MESH 2
&MESH IJK=10,10,10      XB=40.0,60.0,             0.0,20.0,0.0,20.0/ 	 2 m MESH 3
&MESH IJK=10,10,10      XB=60.0,80.0,             0.0,20.0,0.0,20.0/ 	 2 m MESH 4

Note 3: I went through the user guide 'Reading the Fields' in section 20.6 but I did not understand the reason for this error. 
Can you please explain me the reason for this?

Thank you in advance.

**Comment by rmcdermo** [comment, marked as answer]
Please try with latest code (6.9.1).  When I try this little example, it works for me.
```
&HEAD CHID='test'/

&MESH IJK=5,10,10 XB=10.0,20.0, 0.0,20.0,0.0,20.0/ 2 m MESH 1
&MESH IJK=10,10,10 XB=20.0,40.0, 0.0,20.0,0.0,20.0/ 2 m MESH 2
&MESH IJK=10,10,10 XB=40.0,60.0, 0.0,20.0,0.0,20.0/ 2 m MESH 3
&MESH IJK=10,10,10 XB=60.0,80.0, 0.0,20.0,0.0,20.0/ 2 m MESH 4

&TIME T_END=2./

&DUMP DT_UVW=1./

&CSVF UVWFILE='test_uvw_t2_m1.csv'/

&TAIL/
```

**Reply by Amilwickz** [reply]
Thank you very much, Randy! It worked. Appreciate your help.
